packages/metrics/src/Metrics.ts

```diff
@@ -921,6 +939,17 @@ class Metrics extends Utility implements MetricsInterface {
       this.getEnvVarsService().getNamespace()) as string;
   }
 
+  /**
+   * Set the disbaled flag based on the environment variables `POWERTOOLS_METRICS_DISABLED` and `POWERTOOLS_DEV`.
+   */
+  private setDisabled(): void {
+    this.disabled =
+      this.getEnvVarsService().isDevMode() ||
+      this.getEnvVarsService().getMetricsDisabled();
+
+    // TODO if POWERTOOLS_METRICS_DISABLED is set to false, and dev mode is enabled, then POWERTOOLS_METRICS_DISABLED takes precedence and the utility stays enabled.
```

Comment:
Getting this two work requires some changes to `CommonEnvironmentVariablesService`, so I wanted to align first before I start.

The `CommonEnvironmentVariablesService` only provides a `isValueTrue()` method to check for truthy values. That means `POWERTOOLS_METRICS_DISABLED=false` and `POWERTOOLS_METRICS_DISABLED=undefined` (not set) will both evaluate to `false`. However, in order to give `POWERTOOLS_METRICS_DISABLED` precedence over `POWERTOOLS_DEV`, I would need a method to check if `POWERTOOLS_METRICS_DISABLED` is present and not-truthy.


```ts
class EnvironmentVariablesService {

  public isPresent(name: string): boolean {
    return name in process.env;
  }
}
```

If there's another way that I overlooked, please let me know.

Reply:
Good point, I think we could add a new method in `CommonEnvironmentVariablesService` called `isValueFalse()` and use that for the precedence logic.

If you go this route and make changes in the `packages/commons/*` directory, make sure to run `npm run build -w packages/commons` to see them reflected in `packages/metrics/*` (& elsewhere).